src/components/sidebar/portaled-sidebar.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react"
+import { Portal } from "react-portal"
+import { PortalSidebox } from "./styled"
+
+export const PortalSidebar: FC<SidebarProps> = ({ children, right = false }: SidebarProps) => (
+  <Portal>
+    <PortalSidebox shadowXOffset={right ? -2 : 2} side={right ? "right" : "left"}>
```

Comment:
The prop name is kinda hard to understand. What is it for?

Reply:
It is about dropping shadow

Reply:
And why is it 2 or -2 numeric?

Reply:
This exact values from figma

Reply:
@Azein  resolved